crates/astria-sequencer/src/app.rs

```diff
@@ -672,7 +712,7 @@ mod test {
 
     #[tokio::test]
     async fn app_genesis_and_init_chain() {
-        let app = initialize_app(None, vec![]).await;
+        let (app, _storage) = initialize_app(None, vec![]).await;
```

Comment:
make an `initialize_app` and `initiialize_app_with_storage` where `initiialize_app_with_storage` returns the storage but `initialize_app` doesn't (it can just call `initiialize_app_with_storage` and ignore the storage) 

then you can use just `initialize_app` where the storages are ignored